wdtk-client/src/test/java/org/wikidata/wdtk/client/RdfSerializationActionTest.java

```diff
@@ -38,6 +38,7 @@ public void testDefaults() {
 		assertTrue(action instanceof RdfSerializationAction);
 		assertTrue(action.needsSites());
 		assertTrue(action.isReady());
+		assertEquals(action.getActionName(), "RdfSerializationAction");
 	}
 
```

Comment:
Same comment as for JSON, but probably it's enough to test report file writing once since the code is shared.